src/pages/iou/WaypointEditor.js

```diff
@@ -250,7 +250,6 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
                                 state: null,
                             }}
                             predefinedPlaces={recentWaypoints}
-                            resultTypes=""
```

Comment:
Why was this removed? This is important since it allows us to get suggested addresses by the name of a place, business, etc. I created a bug for [this regression](https://github.com/Expensify/App/issues/28936). @DylanDylann @burczu @youssef-lr